composer.json

```diff
@@ -17,16 +17,14 @@
     ],
 
     "require": {
-        "php": ">=5.5.9",
-        "symfony/form": "^2.7|^3.0|^4.0",
```

Comment:
I have noticed a few deprecation notices when running tests on Symfony 5 (like it's in .travis.yml file)
- `The "Symfony\Component\Debug\DebugClassLoader" class is deprecated since Symfony 4.4, use "Symfony\Component\ErrorHandler\DebugClassLoader" instead.`
- `User Deprecated: Loading the file "../src/Controller/" from the global resource directory "/application/Tests/app/src" is deprecated since Symfony 4.4 and will be removed in 5.0.`
- `User Deprecated: Since symfony/validator 4.4: "minMessage" and "maxMessage" are deprecated when the "min" and "max" options are both set. Use "notInRangeMessage" instead.`

and also 1 fatal: 
`PHP Fatal error:  Uncaught Error: Class "Symfony\Component\Debug\Debug" not found in /application/Tests/app/public/index.php:12`

Should we address it?

Reply:
Thank you for your feedback! I am sorry, but I completely changed commit history as I completely reworked it.